sdk/python/tests/compiler/testdata/add_pod_env.yaml

```diff
@@ -25,6 +25,11 @@ spec:
         valueFrom:
           fieldRef:
             fieldPath: metadata.namespace
+      - name: WORKLOW_ID
```

Comment:
Do we want to expose the Workflow ID, which is an Argo-specific concept?

We have RUN_ID, which is expected to be portable (it does not point to an Argo Workflow though.).